docs/docs/general/data-structures.md

```diff
@@ -4,6 +4,9 @@ title: Data structures
 
 The `data` property of a dataset can be passed in various formats. By default, that `data` is parsed using the associated chart type and scales.
 
+The `labels` property of the main `data` property has to contain the same amount of elements as the dataset with the most values. These labels are used to label the index axis (default x axes). The values for the labels have to be provided in an array.
```

Comment:
This isn't strictly true as the labels array can be omitted and instead each data point supplied using objects (see example below). I would caveat this with something like "If the `labels` property is used ..."

```javascript
const data = {
  datsets: [{
    data: [{ x: "a", y: 10}, { x: "b", y: 0}]
  }],
}
```

Reply:
@etimberg https://github.com/chartjs/chartjs-plugin-datalabels/pull/203#discussion_r539458227

Reply:
Thanks @simonbrunel. We should update so that `labels` isn't strictly required in the typing